src/RamOperation.h

```diff
@@ -397,6 +400,10 @@ class RamLookup : public RamSearch {
  * Aggregation
  */
 class RamAggregate : public RamSearch {
+    RamConstValueAnalysis* rcva;
```

Comment:
Should these be created somewhere?

Reply:
Fixed in #866, had to duplicate some code until aggregates get refactored